benchmarks/bench_mnist.py

```diff
@@ -84,11 +87,19 @@ def load_data(dtype=np.float32, order='F'):
     'CART': DecisionTreeClassifier(),
     'ExtraTrees': ExtraTreesClassifier(n_estimators=100),
     'RandomForest': RandomForestClassifier(n_estimators=100),
-    'Nystroem-SVM':
-    make_pipeline(Nystroem(gamma=0.015, n_components=1000), LinearSVC(C=100)),
-    'SampledRBF-SVM':
-    make_pipeline(RBFSampler(gamma=0.015, n_components=1000), LinearSVC(C=100)),
-    'LinearRegression-SAG': LogisticRegression(solver='sag', tol=1e-1, C=1e4)
+    'Nystroem-SVM': make_pipeline(
+        Nystroem(gamma=0.015, n_components=1000), LinearSVC(C=100)),
+    'SampledRBF-SVM': make_pipeline(
+        RBFSampler(gamma=0.015, n_components=1000), LinearSVC(C=100)),
+    'LinearRegression-SAG': LogisticRegression(solver='sag', tol=1e-1, C=1e4),
+    'MultilayerPerceptron': MLPClassifier(
```

Comment:
What do you mean by failing to converge? It runs for all of 400 epochs? I guess that was the idea before, which is why it took so long. I'd be happy with either leaving as is or just having adam in there.